libs/design/src/atoms/form/radio/radio.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { ReactiveFormsModule, FormGroup, FormControl } from '@angular/forms';
+import { DaffRadioModule } from './radio.module';
+import { DaffRadioComponent } from './radio.component';
+
+
+@Component({
+  template: `
+    <daff-radio name="test" value="testValue"></daff-radio>
+  `
+})
+class RadioWrapperComponent { }
+
+@Component({
+  template: `
+ 
+<daff-radio-set [formGroup]="radioGroup" name="fruit">
+  <daff-radio formControlName="fruit" value="apple">Apple</daff-radio>
+  <daff-radio formControlName="fruit" value="grape">Grape</daff-radio>
+  <daff-radio formControlName="fruit" value="peach">Peach</daff-radio>
+</daff-radio-set>
+  `
+})
+class RadioEmbeddedComponent {
+  radioGroup = new FormGroup({
+    fruit: new FormControl()
+  });
+  disable() {
+    this.radioGroup.disable();
+  }
+  setValue() {
+    this.radioGroup.setValue({ fruit: 'pear' });
+  }
+}
+
+describe('DaffRadioComponent', () => {
```

Comment:
Given that this spec covers different things, can you split them out into a `specs` folder with different files. 

The main spec `radio.spec.ts` covers the basics of the element - inputs/outputs/defaults.

The other specs cover: 
* `ControlValueAccessor`
* Usage with `DaffRadioSet`
* Accessibility